packages/features/eventtypes/components/CreateEventTypeDialog.tsx

```diff
@@ -166,76 +187,70 @@ export default function CreateEventTypeDialog({
         enableOverflow
         title={teamId ? t("add_new_team_event_type") : t("add_new_event_type")}
         description={t("new_event_type_to_book_description")}>
-        <Form
-          form={form}
-          handleSubmit={(values) => {
-            createMutation.mutate(values);
-          }}>
-          <div className="mt-3 space-y-6 pb-11">
-            {teamId && (
+        {!teamId ? (
```

Comment:
nit: I personally find it easier to understand if in a conditional statement, we have the shorter branch written earlier. If it doesn't make sense to you, feel free to not make this change :)

So, it would become as follows. It is kind of like early return.
```suggestion
        {teamId ? (
          <TeamEventTypeForm
            isAdmin={isAdmin}
            teamId={teamId}
            submitButton={submitButton}
            handleSuccessMutation={handleSuccessMutation}
          />
        ): ......}
```